forge-gui/res/cardsfolder/upcoming/filigree_vector.txt

```diff
@@ -0,0 +1,13 @@
+Name:Filigree Vector
+ManaCost:3 W
+Types:Artifact Creature Phyrexian Construct
+PT:1/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerDescription$ When CARDNAME enters the battlefield, put a +1/+1 counter on each of any number of target creatures and a charge counter on each of any number of target artifacts.
+SVar:TrigCounter:DB$ PutCounter | ValidTgts$ Creature | TgtPrompt$ Select any number of target creatures to distribute +1/+1 counters to | CounterType$ P1P1 | CounterNum$ 1 | TargetMin$ 0 | TargetMax$ X | SubAbility$ DBChargeCounter
+SVar:DBChargeCounter:DB$ PutCounter | ValidTgts$ Artifact | TgtPrompt$ Select any number of target artifacts to distribute charge counters to | CounterType$ CHARGE | CounterNum$ 1 | TargetMin$ 0 | TargetMax$ Y
```

Comment:
i need to check the rules if both counters are put on them at the same time for cards that trigger of counters

Reply:
i didn't explicit checked rules yet, but my gut is telling me that both types of counters needs to be put on the cards at the same time

Reply:
So the current version needs to be changed or is it correct ?

Reply:
Right now you've got 2 PutCounter lines, so your P1P1 counters are put first (possibly triggering "CounterAdded" triggers) then your CHARGE counters are put...

I feel like we may have something like this wired up already, but I'm not sure.

Reply:
Brokers Ascendancy and a few Ajani PW cards use the params you want, but in a non targeted way. I think you could rig this with a few pump lines for targeting and some remembering (maybe imprinting), then a PutCounterAll line with ValidCards2/CounterType2

Reply:
Thanks I'll give it a try